### DISCUSSION TITLE
Как правильно зарегистрировать приложение в oauth.yandex.ru

### DISCUSSION THREAD

**Discussion by NikcN22** [MarshalX/yandex-music-api#584] [Q&A]
Добрый день.

Токен удалось получить. Запрос `acoount_status` выдаёт данные как будто у пользователя нет подписки:

```
{
   "invocationInfo":{
      "hostname":"music-stable-back-vla-22.vla.yp-c.yandex.net",
      "req-id":"1675796035174453-8726253753145548120",
      "exec-duration-millis":"0"
   },
   "result":{
      "account":{
         "now":"2023-02-07T21:53:55+03:00",
         "region":225,
         "serviceAvailable":true
      },
      "permissions":{
         "until":"2023-02-07T18:53:55+00:00",
         "values":[
            "landing-play",
            "feed-play",
            "mix-play"
         ],
         "default":[
            "landing-play",
            "feed-play",
            "mix-play"
         ]
      },
      "bar-below":{
         "alertId":"xxx",
         "text":"тест Подписка скоро закончится. А вместе с ней закончатся музыка и подкасты без ограничений. Не надо так.",
         "bgColor":"#FF5038",
         "textColor":"#FFFFFF",
         "alertType":"Churn_SubscriptionEnd_Music",
         "button":{
            "text":"Оформить подписку",
            "bgColor":"#FFFFFF",
            "textColor":"#191919",
            "uri":"https://plus.yandex.ru/?source=music_web_churn_subscriptionend"
         },
         "closeButton":false,
         "closeButtonStyles":{
            
         }
      }
   }
}
```

request выглядит так:

**request URL** - Optional(https://api.music.yandex.net/account/status)
**request HEASERS** - 
([
   "Accept-Language":"ru",
   "X-Yandex-Music-Client":"Name-native-macOS-app-for-Yandex-Music",
   "Authorization":"OAuth y0_AgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
])
49 Optional([
   "Accept-Language":"ru",
   "Authorization":"OAuth y0_AgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
   "X-Yandex-Music-Client":"Name-native-macOS-app-for-Yandex-Music"
])
**request METHOD** -Optional("GET")

В чём может быть проблема? Подписка у пользователя для которого получен токен есть.

**Comment by MarshalX** [comment]
https://github.com/MarshalX/yandex-music-api/discussions/513#discussion-3903521
